src/request/plan.rs

```diff
@@ -63,6 +64,7 @@ pub struct RetryableMultiRegion<P: Plan, PdC: PdClient> {
     pub(super) inner: P,
     pub pd_client: Arc<PdC>,
     pub backoff: Backoff,
+    pub preserve_results: bool,
```

Comment:
I'm thinking whether there is a better name, since "result" is too generic and can be ambiguous. Or at least we can add some comment to explain.